groupProject1/script.js

```diff
@@ -0,0 +1,39 @@
+//Harvard Art Museum API Key
+var APIKey= "9110745d-175a-40d6-badc-2106d0abd90b";
+
+
+function searchArt(event) {
+    //prevent default refresh
+    event.preventDefault();
+    
+    var queryURL = "https://api.harvardartmuseums.org/object?q=totalpageviews:0&size=10&apikey=" + APIKey;
+
+
+    // If you comment out line 11 and replace it with the line below, the console log will show data on all 255 cultures available to choose from. We should pick maybe 10, each of which have a decent amount of art (which I think is shown in "objects")
+
+   /*  var queryURL= "https://api.harvardartmuseums.org/culture?size=255&apikey=" + APIKey; */
+
+    //fetch request
+    fetch(queryURL)
+        .then(function(response){
+            return response.json();
+        })
+        .then(function(data) {
+            console.log(data);
+        });
+};
+
+//need a function to display search history and clear current search
+
+//when search button is clicked...
+search.addEventListener("click", searchArt);
+
+/*TODO
```

Comment:
Here is the TODO list I mentioned. This is what I was planning to work on next. If anyone wants to sync up and work together on it, please hit me up on Slack.